### ISSUE TITLE
Special tokens representation

### ISSUE THREAD

**Issue by UriLawgeex** [CyberZHG/keras-bert#15]
In bert.py, the special tokens in base_dict use angle brackets, but in the pretrained bert models released by Google, the vocab file has them with rectangular brackets. _e.g._, your TOKEN_UNK is '\<UNK\>' but in the vocab file it is '[UNK]'. Also, your padding token is an empty string while the vocab file's padding token is '[PAD]'. This causes problems when trying to adapt your bert_fit_demo to continue training the released models, instead of training a model from scratch.

**Comment by CyberZHG** [comment]
I think in this case you should use the vocabulary in the pre-trained model directly instead of building the dictionary from scratch.

**Comment by UriLawgeex** [comment]
Thanks!
